quickrun.el

```diff
@@ -775,7 +775,12 @@ if you set your own language configuration.")
         (unless (file-directory-p formatted)
           (throw 'quickrun
                  (format "'%s' is not existed directory" it)))
-        (setq quickrun-option-default-directory formatted)))))
+        (let* ((has-space (string-match " " formatted))
```

Comment:
I might use `string-match-p` instead here. I think you get the same result but as I aware people use `-p` mostly for boolean. 😕 Just to match the standard! And the first argument take the regular expression, hence you may do it with `[ \t]` instead but of course this isn't necessary because I don't think a path cannot contain tab character. Otherwise seems to be good! 😄

Reply:
Will change to use `string-match-p`. I also checked that `\t` is a valid character in path names, so I will add that to the regex